include/aspect/particle/world.h

```diff
@@ -122,11 +122,14 @@ namespace aspect
         get_particles() const;
 
         /**
-         * Returns the number of particles in the cell that contains the
-         * most tracers in the global model.
+         * Calculates and stores the number of particles in the cell that
+         * contains the most tracers in the global model. This variable is a
+         * state variable, because it is needed to serialize and deserialize
+         * the particle data correctly in parallel (it determines the size of
+         * the data chunks per cell that are stored and read).
          */
```

Comment:
can you add whether this state variable is always kept up to date? I guess the function is also public, which however no longer makes much sense since no external entity should call it I suppose.